micropather.h

```diff
@@ -391,8 +391,8 @@ namespace micropather
 		const Item* Find( void* start, void* end );
 		
 		Item*	mem;
-		int		allocated;
-		int		nItems;
+		unsigned	allocated;
+		unsigned	nItems;

```

Comment:
So this is probably the wrong solution. A `size_t` value would be much better here, though may require a few more changes.

I don't see much activity on this project, so not sure if there is any interest in fixing this. If so, I'd be happy to rework and submit a more proper fix.